monitoring/mongodb/alerts.yaml

```
@@ -139,3 +139,45 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      count_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1h]) > 0
+    for: 1h
+    labels:
+      severity: warning
+    annotations:
+      description: "Mongodb pod `{{ $labels.pod }}` has been in the 'RECOVERING' state at least once in the past hour. The instance may not be able to join the replica set if the platform ingests a large number of operations during this time. This alert is expected if the 'Resync a Data Services MongoDB Member' procedure has recently been executed."
+      summary: MongoDB node recovering for too long
+
+  - alert: MongoDbInvalidState
+    expr: |
+      avg_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state=~"6|8|10"}[1m]) > 0
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      description: "Mongodb pod `{{ $labels.pod }}` is in the `{{ $labels.rs_state }}` state. Immediate attention is required."
```

```suggestion
      description: "MongoDB pod `{{ $labels.pod }}` is in the `{{ $labels.rs_state }}` state. Immediate attention is required."
```

(applies to everything, let's follow the real naming of MongoDB)